frontends/web/src/routes/bitsurance/dashboard.tsx

```diff
@@ -160,7 +163,7 @@ export const BitsuranceDashboard = ({ accounts }: TProps) => {
                     </div>
 
                   </div>
-                ) : null)}
+                ) : null) : <Skeleton />}
```

Comment:
Very nice 👍 I think this skeleton / loading component could be improved so that it's clearer to the users that it's a loading component.

I could think of 2 things rn. Either:

- Make the skeleton taller (more or less as tall as the account container box thingy), or
- use our spinner component. I tried using  `HorizontallyCenteredSpinner` and looked fine.

Reply:
Thanks, I switched to the `HorizontallyCenteredSpinner`